README.md

```diff
@@ -46,3 +46,16 @@ jobs:
     - NOTE: when not specified, the `latest` version will be installed
     - NOTE: the Phylum CLI 2.0.0 release changed the way the artifacts are packaged and released, which means this
             option should **NOT specify a version less than 2.0.0**
+
+### License
+
+Copyright (C) 2022 Phylum, Inc. - All Rights Reserved
```

Comment:
I am not a lawyer, but my understanding is that "All Rights Reserved" is just unnecessary text here. And personally, I find it confusing since we are intentionally granting rights via the GPL.

Also, as I have recently noticed, it seems to be appropriate to use 2 spaces between the year and the author (but I couldn't tell you why...).

```suggestion
Copyright (C) 2022  Phylum, Inc.
```